docs/CONTRIBUTING.md

```diff
@@ -84,23 +84,22 @@ Here are these steps you should follow to get your dev environment setup:
 
 1. `git clone https://github.com/cdr/code-server.git` - Clone `code-server`
 2. `git clone https://github.com/cdr/vscode.git` - Clone `vscode`
-3. `cd vscode && git checkout code-server-v2` - checkout the branch we use (not the default)
-4. `cd vscode && yarn install` - install the dependencies in the `vscode` repo
-5. `cd code-server && yarn install` - install the dependencies in the `code-server` repo
```

Comment:
🧹 We're now using the `main` branch (instead of `code-server-v2` so step 3 was removed.